From 91692211404172dfd9d1e210c1f7cf1c3b064f27 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Sat, 30 May 2020 16:17:37 -0400
Subject: [PATCH 1565/1936] Resolve Windows build issue

Credit to tracking down the issue goes to FormulaGX at
https://github.com/zynaddsubfx/zyn-fusion-build/issues/52
---
 src/Misc/MiddleWare.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Misc/MiddleWare.cpp b/src/Misc/MiddleWare.cpp
index 740fd1cf..ea7f18ca 100644
--- a/src/Misc/MiddleWare.cpp
+++ b/src/Misc/MiddleWare.cpp
@@ -190,7 +190,7 @@ void preparePadSynth(string path, PADnoteParameters *p, rtosc::RtData &d)
 
 #ifdef WIN32
     unsigned num = p->sampleGenerator([&path,&d]
-                       (unsigned N, PADnoteParameters::Sample &s)
+                       (unsigned N, PADnoteParameters::Sample &&s)
                        {
                            //printf("sending info to '%s'\n",
                            //       (path+to_s(N)).c_str());
-- 
2.47.0

